website/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="../src/bottomsheet.css" />
+  </head>
+
+  <body>
+    <div class="carousel">
+      <div class="">
+        <h4 class="series">Comedy Series</h4>
+        <ul class="scroll-snap-slider"></ul>
+      </div>
+      <!-- <a href="./maps.html">link</a> -->
+    </div>
+    <script type="module" src="index.js"></script>
```

Comment:
```suggestion
    <script type="module" src="/index.js"></script>
```

Vite needs this `/` to resolve the file